packages/vue/src/components/FModal/FModal.vue

```diff
@@ -119,6 +119,15 @@ export default defineComponent({
                 return sizes.includes(value);
             },
         },
+        /**
+         * Default behavior is that the modal will restore focus to previous element once closed.
+         * This boolean could be used if you in the application needs to set focus to another element.
+         */
+        disableCloseFocus: {
```

Comment:
Som ett alternativ till `disable-close-focus` skulle jag säga bara `focus` och med fyra värden:

- `"off"` - ingen fokushantering.
- `"on"` (default) - fokushantering både vid öppning och stängning.
- `"open"` - fokuserar bara vid öppning.
- `"close"` - fokuserar bara vid stängning.

För att underlägga för konsumenten kan vi också acceptera `true` och `false` som alias för `"on"` och `"off"`.

Motiveringar:

* Propen är inte återanvändningsbar för att stänga av initialt fokus vilket är något vi vill kolla på i framtidien framförallt för `FFormModal` där konsumenten vill ha mer kontroll.
* Negationer kräver mer kognitiv komplexitet att resonera runt, det är kanske ingen jättestor grej här men blir snabbt ohållbart om man tänker sig att du har en positiv flagga i komponenten `:disable-close-focus="!useFocus"`.
* Blir mer likt standard HTML attribut, ta [autocomplete](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) som exempel. 
* `disable-close-focus` är ganska långt och verbose, rent allmänt föredrar jag kortare namn speciellt när resten av orden inte tillför något mer än brus.

Reply:
Hade det varit ett alternativ att jag i detta läget enbart inför `focus=open`? Eller åtminstonde inte alla delarna. Tänker att det kan växa fram genom flera prs isåfall.

Tycker detta api'et såhär blir mycket bättre ut mot konsumenter. Som sagt, såg mitt enbart som ett lösningsförslag att diskutera kring, så bara skjut ned att 💣

Reply:
PR omskriven till att stödja on, off och open.

Reply:
@dvitamin @MCFK @NicklasWei ^